### ISSUE TITLE
Hard to see the check-box of package in ‘Updates’ Tab is focused with a dash-line box when navigating through Tab in Blue/Blue (Extra Contrast)/Light themes

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#8963]
Related https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1244648

## Details about Problem
NuGet version: NuGet Client Dev\5.5.0.6353
VS Version: Master\29619.96
OS: Windows-10-Enterprise-RS5
## Detailed repro steps 
1. Launch VS and click “Create a New Project” button.
2. Create an ASP.NET Web Application (.NET Framework) project (Single Page Application).
3. Open PM UI of this project and go to ‘Updates’ tab.
4. Navigate by Tab to the package list.
5. There is a dash-line showing on the outer edge of the first package.
6. Hit Tab again, attempt to focus on the check-box of this package.
## Expected
Easy to see the check-box of package in ‘Updates’ Tab is focused with a dash-line box. 
## Actual
Hard to see the check-box of package in ‘Updates’ Tab is focused with a dash-line box as below screenshot. Note: actually it is focused since you can see the “√” when hitting Space on keyboard.
![CannotSeeDashLineBox](https://user-images.githubusercontent.com/20632625/71239864-91412200-2342-11ea-9f5e-a25e991c32ea.png)
## Note
1. It repro on 16.2 GA (D16.2\29116.123) with implicit NuGet 5.2.0.6090.
2. It doesn’t repro in Dark theme.
3. Resolution suggested: make the bash-box a little bigger than the check-box.

**Comment by dominoFire** [comment]
Confirmed in 29619.127

**Comment by v-luzh** [comment]
Same issue happen when focusing on the "Package sources"/"Version"/"Dependency behavior"/"File conflict action" drop-down box in PM UI as below screenshot.
![image](https://user-images.githubusercontent.com/20632625/98085207-f6005f00-1eb7-11eb-81f6-76797d6ab940.png)

**Comment by donnie-msft** [comment]
Related https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1244648

**Comment by donnie-msft** [comment]
> Same issue happen when focusing on the "Package sources"/"Version"/"Dependency behavior"/"File conflict action" drop-down box in PM UI as below screenshot.
> ![image](https://user-images.githubusercontent.com/20632625/98085207-f6005f00-1eb7-11eb-81f6-76797d6ab940.png)

@v-luzh Thanks for adding that, and I agree with you that this isn't great focus behavior. 
However, I believe this is something WPF will need to address in the future. The ComboBox controls probably do not meet https://w3c.github.io/wcag/understanding/focus-appearance-minimum which are not required by Microsoft today per the Accessibility team.  For now, I will not track this in our repo since it appears to be out of scope.
